src/test/java/com/volasoftware/tinder/UserServiceTest.java

```diff
@@ -138,4 +141,22 @@ public void testLoginUser() {
 
     verify(userRepository,times(1)).findOneByEmail(authentication.getName());
   }
+
+  @Test
+  public void testReplaceOldPassword() throws MessagingException, IOException {
+    String email = "test@test.com";
+    String password = passwordGenerator.generatePassword();
+
+    User user = new User();
+    user.setEmail(email);
+    user.setPassword(password);
+
+    when(userRepository.findOneByEmail(user.getEmail())).thenReturn(Optional.of(user));
+    when(userRepository.save(any(User.class))).thenReturn(user);
+    emailSenderService.sendEmail(
+        "New Password", Collections.singleton(user.getEmail()), user.getPassword());
+
+    verify(emailSenderService, times(1)).
+            sendEmail("New Password", Collections.singleton(user.getEmail()),password);
```

Comment:
Not need of this, you're trying to verify the method that you're testing, so no point to verified